conformance/tests/classes_classvar.py

```diff
@@ -57,8 +57,14 @@ class ClassA(Generic[T, P]):
     good1: CV[int] = 1
     good2: ClassVar[list[str]] = []
     good3: ClassVar[Any] = 1
+    # > If an assigned value is available, the type should be inferred as some type
+    # > to which this value is assignable.
+    # Here, type checkers could infer good4 as `float` or `Any`, for example.
     good4: ClassVar = 3.1
-    good5: Annotated[ClassVar[list[int]], ""] = []
+    # > If the `ClassVar` qualifier is used without any assigned value, the type
+    # > should be inferred as `Any`:
+    good5: ClassVar
```

Comment:
I think we can use a `# E?` here to permit type checkers to optionally error on this as an uninitialized ClassVar.